### PR TITLE
Restore cron iden in storm logs, add httpapi idens to storm logs (SYN-7260)

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: synapse/lib/snap.py [hunk 1]
```diff
@@ -717,7 +717,11 @@ async def iterStormPodes(self, text, opts, user=None):
 
         show_storage = False
 
-        self.core._logStormQuery(text, user, info={'mode': opts.get('mode', 'storm'), 'view': self.view.iden})
+        info = opts.get('_loginfo')
+        if info is None:
+            info = {}
```
Comment (MichaelSquires) [inline synapse/lib/snap.py:722]:
```suggestion
        info = opts.get('_loginfo', {})
```

File: synapse/lib/view.py [hunk 2]
```diff
@@ -868,20 +868,18 @@ async def _initViewLayers(self):
...
-        if log_info is None:
-            log_info = {}
-
-        log_info['mode'] = opts.get('mode', 'storm')
-        log_info['view'] = self.iden
-
-        self.core._logStormQuery(text, user, info=log_info)
+        info = opts.get('_loginfo')
+        if info is None:
+            info = {}
```
Comment (MichaelSquires) [inline synapse/lib/view.py:880]:
```suggestion
        info = opts.get('_loginfo', {})
```

File: synapse/lib/view.py [hunk 3]
```diff
@@ -997,8 +995,11 @@ async def runStorm():
                                 count += 1
 
                         else:
-                            self.core._logStormQuery(text, user,
-                                                     info={'mode': opts.get('mode', 'storm'), 'view': self.iden})
+                            info = opts.get('_loginfo')
+                            if info is None:
+                                info = {}
```
Comment (MichaelSquires) [inline synapse/lib/view.py:1000]:
```suggestion
                            info = opts.get('_loginfo', {})
```

**Review by MichaelSquires — COMMENTED** [review]
minor suggestions

**Review by Cisphyx — DISMISSED** [review]

**Review by MichaelSquires — APPROVED** [review]

**Review by Cisphyx — APPROVED** [review]

**Pull request merged**